x-pack/plugins/file_upload/server/telemetry/telemetry.ts

```diff
@@ -74,8 +74,8 @@ export async function updateTelemetry(internalRepo?: ISavedObjectsRepository) {
 
 function incrementCounts(telemetry: Telemetry) {
```

Comment:
Any reason not using `SavedObjectsRepository.incrementCounter`?

Recently we've added support to increment multiple fields with custom increments:

```
incrementCounter<T = unknown>(type: string, id: string, counterFields: Array<string | SavedObjectsIncrementCounterField>, options?: SavedObjectsIncrementCounterOptions): Promise<SavedObject<T>>;
```

Reply:
@Bamieh This PR is migrating code from ML to a separate plugin, trying to change as little as possible. I can create an issue to track using SavedObjectsRepository.incrementCounter but would like to leave any refactoring out of this initial file_upload creation pull request.